committee-steering/governance/annual-reports.md

```diff
@@ -21,83 +21,87 @@ should be highlighted
 ## Reporting Process  
 
 Chairs and Organizers are responsible for compiling a yearly public report but
-may be completed with the help of members of that group. Chairs and Organizers
+may be completed with the help of members of that group. Groups are encouraged
+to make this an agenda item of topic to discuss together. Chairs and Organizers
 ensure that reports are complete, accurate, and submitted to the Steering
 Committee.
 
+### 2021 and on:
+1. Steering Committee liaison reaches out to group leads to kick off
+2. Chairs/Organizers will:
+- share with their groups (see tips below) and begin to 
+compile the doc due at the end of February 
+- use this [question template]
+- submit a pull request to the groups documentation in the community repo
+- if there are private matters to address with Steering before submitting the 
+pull request, share a copy with your laison and wait until comments are
+addressed before submitting.
+3. The March Chairs, Tech Leads, and Organizers meetings will be used as follow 
```

Comment:
I spent a minute trying to figure out what "March Chairs" were... :-P

maybe `The March "Chairs, Tech Leads, and Organizers" meetings...`?